common/pulp/common/error_codes.py

```diff
@@ -113,6 +113,9 @@
 PLP0043 = Error("PLP0043", _("Database 'write_concern' config can only be 'majority' or 'all'. "
                              "Refer to /etc/pulp/server.conf for proper use."), [])
 PLP0044 = Error("PLP0033", _("The target importer does not support the types from the source"), [])
+PLP0045 = Error("PLP0045", _("The importer %(importer_id)s indicated a failed response when "
+                             "uploading %(unit_type)s unit to repository %(repository_id)s."),
+                ['importer_id', 'unit_type', 'repository_id'])
```

Comment:
One more thought - we often do one commit per pull request but that's not written in stone. In order to be more atomic, you could fix the incorrect number in one commit, and then rebase the rest of your change as a second commit. Then it's nice and clearly separated changes, and they can go into one pull request.

In my opinion, it's commits that need to be atomic, not pull requests. Pull requests should *usually* be atomic, but there are times it makes sense to bring two changes together and this is an example of one of those times.

It's up to you though, I think either path is fine.